core/services/wifi/WifiManager.py

```diff
@@ -135,12 +135,12 @@ async def add_network(self, credentials: WifiCredentials) -> int:
         try:
             data = await self.wpa.send_command_add_network()
             data = data.strip()
-            if data == b"Fail":
+            if not data.isdigit():
                 raise RuntimeError("Failed to add new network.")
 
-            network_number = data.decode("utf-8")
-            await self.wpa.send_command_set_network(str(network_number), "ssid", f'"{credentials.ssid}"')
```

Comment:
Is this in the wrong commit ? Should be a fixup, check my previous comment

Reply:
This part is not, but the previous one is. Will change there.

Reply:
Fixed.